nvflare/private/fed/server/training_cmds.py

```diff
@@ -320,7 +320,7 @@ def _process_client_status_replies(self, conn, replies):
             else:
                 table.add_row([client_name, app_name, job_id, "No Reply"])
 
-    def _add_scope_info(self, table, site_name, scope_names: List[str], default_scope: str):
+    def _add_scope_info(self, table, site_name, scope_names: [str], default_scope: str):
```

Comment:
let's keep the type hint

Reply:
The List[str] and [str] essentially the same. It's a string list.

Reply:
From type hinting tool static analysis perspective (for example "mypy"),
Only List[str] is valid for the check.

otherwise we get
error: Bracketed expression "[...]" is not valid as a type  [valid-type]